modules/gapi/test/cpu/gapi_core_tests_cpu.cpp

```diff
@@ -11,7 +11,7 @@
 
 namespace
 {
-    #define CORE_CPU [] () { return cv::compile_args(cv::gapi::core::cpu::kernels()); }
+const auto getArgsCoreCpu = [] () { return cv::compile_args(cv::gapi::core::cpu::kernels()); };
```

Comment:
I belive the old naming was better. It least you see clearly that this test is to run on CPU (Fluid, OCL, etc).
Now one should care what `getArgsCoreCpu` means.

Reply:
reverted to previous version